compel/plugins/std/log.c

```diff
@@ -355,6 +355,26 @@ void print_on_level(unsigned int loglevel, const char *format, ...)
 	sbuf_log_flush(&b);
 }
 
+int flog_encode_msg(int loglevel, unsigned int nargs, unsigned int mask, const char *format, ...){
+	/*TO DO: Change this to add binary logging here*/
```

Comment:
@prakritigoyal19 , you have commented-out all the compel's messages, why is flog needed in compel after it?

Reply:
While compiling the code I am getting the error: external symbol in pie. This causes the build to fail with an error 255. To overcome this error, I added this to compel. I will look for alternatives to be able to fix this. 

Also, the rpc test of criu tries to read one of the messages written by compel. The message is: “The criu itself is within dumped tree”, from criu/compel/src/lib/infect.c. This causes the Travis CI to fail for multiple architectures after I comment out all the compel messages. Please let me know if I should allow only that message to be printed or should I try changing the message in rpc or would there be some other alternative to overcome this error.